.github/workflows/release-binaries.yml

```diff
@@ -44,14 +44,21 @@ jobs:
       upload: ${{ steps.vars.outputs.upload }}
 
     steps:
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+            python3-github
```

Comment:
I'm thinking we should probably install from a `requirements.txt` so that we get better version control? `llvm/utils/git/requirements.txt` would work I think.